website/docs/r/volume.html.markdown

```diff
@@ -62,6 +62,106 @@ resource "linode_volume" "foobar" {
 }
 ```
 
+If you use a `linode_instance_config` you should not include the `linode_id` in the `linode_volume`. The volume will be attached to the linode via the instance config. The folowing is a complete example which demonstrates the usage:
+
+```hcl
+terraform {
+  required_providers {
+    linode = {
+      source  = "linode/linode"
+      version = "2.5.2"
+    }
+  }
+}
+
+provider "linode" {
+  # use default LINODE_TOKEN env var
+}
+
```

Comment:
I think we can drop this bit since it's covered in the top-level provider documentation

Reply:
I think this would be fine though we would want to note above that this is no longer a complete working example.